scss/_patterns_lists.scss

```
@@ -333,6 +333,10 @@ $list-status-icon-nudge: 0.3rem; // 0.3rem seems to be a magic number that posit
         width: $bullet-width;
       }
     }
+    .p-strip--dark &::before {
+      background-color: $color-x-dark;
```

I guess let's use the same as dark strip

```suggestion
      background-color: $color-dark;
```